module-2/backend/week_two.md

```diff
@@ -5,37 +5,93 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+ - ActiveRecord is a way to use Ruby to talk to databases. It lets you abstract out database logic to more readable, easily usable ruby methods.
 2. What is a migration?
+  - It's when a bunch of geese... I mean it's a file you can use when need to create or update or delete tables in a database. 
 3. How does a table relate to a model?
+  - A table holds the raw data that the model's logic can iterate over.
 4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+  - belongs_to and has_many are class methods. As an example: a library has_many books, while a book belongs_to a library.
```

Comment:
It's tricky, but these are actually instance methods. Think of it this way, you need to have a specific *instance* of a library before you can use the methods that `has_many`/`belongs_to` have to offer us. For example:

```ruby
# assuming our Library class `has_many` books.
denver_public = Library.find(1)
denver_books = denver_public.books
```